### ISSUE TITLE
video display (Trac #36)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#40]
-create widget for viewing previous experiments (can adjust frame to
correspond to a specific protocol step)
-enable drawing of the video on the main window canvas with the device overlayed. This will require some form of registration (i.e.,
transforming the video or device geometry so that they line up)

Migrated from http://microfluidics.utoronto.ca/ticket/36
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "-create widget for viewing previous experiments (can adjust frame to\ncorrespond to a specific protocol step)\n-enable drawing of the video on the main window canvas with the device overlayed. This will require some form of registration (i.e.,\ntransforming the video or device geometry so that they line up)", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "video display", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-06T21:41:17", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120130T08:34:27: christian@fobel.net commented:

> commit 99287f7adc5f553c622ccbd47d0722389a926915
> Author: Christian Fobel <christian@fobel.net>
> Date:   Mon Jan 30 03:14:37 2012 -0500
> 
>     [SUBMODULE] Update opencv module, degrade back...
>     
>     [SUBMODULE] Update opencv module.
>     Degrade back to old behaviour (rather than video overlay) when camera cannot be
>     connect to.  See #40.

**Comment by cfobel** [comment]
Trac update at 20120130T08:34:30: christian@fobel.net commented:

> commit 670c04d460b9586818cded94c6bb5b750284c553
> Author: Christian Fobel <christian@fobel.net>
> Date:   Mon Jan 30 03:14:37 2012 -0500
> 
>     [SUBMODULE] Update opencv module, add device ov...
>     
>     [SUBMODULE] Update opencv module.
>     Add device overlay, See #40.

**Comment by cfobel** [comment]
Trac update at 20120211T21:58:28: 
* christian@fobel.net commented:

> commit 4068748026901f1cf2d9ae4d0958c6c0b8b450d5
> Author: Christian Fobel <christian@fobel.net>
> Date:   Sat Feb 11 16:41:51 2012 -0500
> 
>     [SUBMODULE] Update opencv and add working devi...
>     
>     [SUBMODULE] Update opencv.
>     Add working device image registration dialog. Fixes #40.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
